### PR TITLE
add nodereinstall command

### PR DESCRIPTION
**Pull request by brock**
The `nodereinstall` command completely re-installs node, npm and nvm. See the article on [Stack Overflow](http://stackoverflow.com/a/11178106/2083544) that got things started, and the [gist](https://gist.github.com/brock/5b1b70590e1171c4ab54) that led me to create this.

### CODE REVIEW

File: _posts/2015-03-04-nodereinstall.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: "nodereinstall"
+description: "Complete Node Reinstall. This deletes everything, yes everything, and re-installs Node and NPM with NVM, then re-installs global NPM modules."
+category: nodejs
+tags: [osx, nodejs]
+---
+This package re-installs Node.js, NPM and NVM.
+## Usage
```
Comment (yorkie) [inline _posts/2015-03-04-nodereinstall.md]: Add a new line before the list? just like the raw source https://raw.githubusercontent.com/bpkg/bpkg.github.io/master/_posts/2014-06-01-term.md, even though it looks good at github documentation :)

File: _posts/2015-03-04-nodereinstall.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+`nodereinstall`
+## Links
```
Comment (yorkie) [inline _posts/2015-03-04-nodereinstall.md]: ditto

**Comment by yorkie** [comment]
LGTM but few nits :+1:

**Comment by jwerle** [comment]
+1 after the small changes suggested by @yorkie

**Comment by brock** [comment]
Let me know if that's what you guys meant. Thanks!

**Comment by jwerle** [comment]
@brock looks good
See https://github.com/brock/node-reinstall/pull/1

**Comment by brock** [comment]
Thanks @jwerle I was in the process of updating with a package.json and a Makefile via https://github.com/brock/node-reinstall/pull/2

Will get the README updated too

**Comment by yorkie** [comment]
oh @brock would you please take a look at this page: http://www.bpkg.io/pkg/nodereinstall/, i guess the diff should be:

```diff
+---
+layout: post
+title: "nodereinstall"
+description: "Complete Node Reinstall. This deletes everything, yes everything, and re-installs Node and NPM with NVM, then re-installs global NPM modules."
+category: nodejs
+tags: [osx, nodejs]
+---
+
+This package re-installs Node.js, NPM and NVM.
+
+## Usage
+```
+$ nodereinstall
+```
+
+## Links
+
+* [Source Code (Github)](https://github.com/brock/node-reinstall)
+* [Author: Brock Angelo](https://github.com/brock)
+
```

**Comment by brock** [comment]
@yorkie See https://github.com/bpkg/bpkg.github.io/pull/11

**Comment by yorkie** [comment]
Thanks @brock http://www.bpkg.io/pkg/nodereinstall/ is working :)

**Pull request merged**